Automattic/sensei
# Add quiz support for course theme

Fixes #4500 
Fixes #4428

### Changes proposed in this Pull Request

* Add a `single-quiz` template that's used for taking the quiz
* Add a `sensei-lms/course-content` that renders the current page content, including post-type specific logic (lesson and quiz)
* Add quiz rendering to this block using the same question loop and hooks as the normal quiz template.

### Testing instructions

* Enable the feature flag: `add_filter( 'sensei_feature_flag_course_theme', '__return_true' );`
* Open and take a course with course theme enabled that has quizzes. 
* Open a lesson quiz. Make sure it shows up in the quiz focus view.
* Complete the quiz and make sure it gets graded. View the quiz again as the learner, make sure it shows up inside the course theme view.


### Screenshot / Video

#### Quiz 'focus' page:

<img width="1217" alt="image" src="https://user-images.githubusercontent.com/176949/144323602-9cb65df1-98b7-4301-828a-05b7806dad73.png">

#### Completed quiz inside the normal layout:
<img width="1232" alt="image" src="https://user-images.githubusercontent.com/176949/144323659-11812032-ac2a-478c-b023-cc1f90766d68.png">
